docs/conf.py

```diff
@@ -91,7 +91,7 @@
     .. note::
         This page was generated from `docs/{{ docname }}`__.
 
-        __"""
+    __ https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/docs/{{ docname }}"""
```

Comment:
@kt474 It won't be always accurate if we point to the main branch link. We need to point to a stable branch or a tag. So if we do something like https://github.com/Qiskit/qiskit-ibm-runtime/tree/{{ release }}/docs/{{ docname }} would it work? Can you test if we can use the `release` variable from this file?

Reply:
Yeah we can do something similar to whats done in [qiskit-nature](https://github.com/Qiskit/qiskit-nature/blob/stable/0.3/docs/conf.py) like this 

```
link_str = f" https://github.com/Qiskit/qiskit-ibm-runtime/blob/{release}/docs/"
nbsphinx_prolog += link_str + "{{ docname }}"
```

where it'll use the release variable (the link still wouldn't work at the moment bc 0.1.0 doesn't exist)